Makefile

```diff
@@ -242,8 +242,12 @@ $(ENVSUBST_BIN): $(ENVSUBST) ## Build envsubst from tools folder.
 $(GOLANGCI_LINT): ## Build golangci-lint from tools folder.
 	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) github.com/golangci/golangci-lint/cmd/golangci-lint $(GOLANGCI_LINT_BIN) $(GOLANGCI_LINT_VER)
 
-$(KUSTOMIZE): ## Build kustomize from tools folder.
-	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) sigs.k8s.io/kustomize/kustomize/v4 $(KUSTOMIZE_BIN) $(KUSTOMIZE_VER)
+$(KUSTOMIZE): ## Build kustomize
```

Comment:
I'm a little confused about what this PR is trying to achieve... I thought the point of #1671 was to make sure all scripts in the repo that use `kubectl` (and call ensure-kubectl.sh) use the makefile to install it and use the version in hack/tools/bin

I think a good example of that is how `apidiff` is installed and used in ci-apidiff.sh. We want the same thing for kubectl. @jsturtevant please correct me if I'm misunderstanding the issue.

Reply:
@CecileRobertMichon Guess even I am confused with the issue. I'll try to figure out.

Reply:
let me know if you need help, James is on vacation this week

Reply:
Sure @CecileRobertMichon but I guess it will need sometime

Reply:
@CecileRobertMichon Is there an example PR I could refer to? I am actually kind of lost with this.